Gemfile

```diff
@@ -22,6 +22,7 @@ gem 'sequel', require: nil
 gem 'sidekiq', require: nil
 gem 'simplecov', require: false, group: :test
 gem 'simplecov-cobertura', require: false, group: :test
+gem 'sneakers', '2.12.0.rc1', require: nil
```

Comment:
This was released on 4 March, 2019. Do you know how far off a GA is? The reason I'm asking is, it would be a pity to add support for Sneakers but only support a rc for an indefinite amount of time.

Reply:
I agree however it doesn't look like they are going to release that soon, you can follow here:
https://github.com/jondot/sneakers/issues/367

Reply:
Gemfile updated